dwitter/feed/views.py

```diff
@@ -9,8 +9,19 @@
 from django.contrib.auth.decorators import login_required
 from django.utils import timezone
 from functools import wraps
+from datetime import datetime, timedelta
+from math import log
 import json
 
+def epoch_seconds(date):
+    epoch = datetime(2015,5,5)
```

Comment:
Just an arbitrary date before dwitter.

I guess I could've gone for 2000,1,1 or something more memorable.